### PR TITLE
[DDW-769] Remove locale-specific rules from the CSS files

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR removes locale-specific rules from the CSS files.

---

## Screenshots

ezgif com-video-to-gif

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1582847292058000)
- [ ] Test on Daedalus Diagnostics length between 'state directory' label and path is not overlapping other content OR breaking into multiple lines when changing Daedalus window size...

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by DeeJayElly** [comment]
> @DeeJayElly there are still some edge cases where the UI falls apart:
> Screenshot 2020-02-28 at 10 20 54
> 
> Can you try removing breakpoints and use flex-box instead?

We need max-width in order to show the ellipsis (...) so i just tried to tweak it now.

**Comment by DeeJayElly** [comment]
> @DeeJayElly rejected! This is what I want: https://css-tricks.com/flexbox-truncated-text/

@tomothespian @nikolaglumac @yakovkaravelov Whole Diagnostic screen is now refactored to use <div> elements and flexbox alignment, table is removed and different media queries for max-width of state directory trimming is now handled in a more cleaner way.

Please review it.

**Comment by nikolaglumac** [comment]
@DmitriiGaico @ManusMcCole @gnpf @IuliaDolishniak please test this one today 🙏

**Comment by nikolaglumac** [comment]
@DeeJayElly the state dir line breaks in an awkward way in Japanese: 
Screenshot 2020-03-09 at 13 41 36

**Comment by nikolaglumac** [comment]
@tomothespian please review this one today 🙏

**Comment by DeeJayElly** [comment]
> @DeeJayElly the statuses 'YES' and 'NO' are no longer color-coded?
> Screenshot 2020-03-09 at 13 37 49
> 
> The "OPEN" button text is no longer vertically centered?
> Screenshot 2020-03-09 at 13 38 36

Both of these are fixed now.

**Comment by nikolaglumac** [comment]
Thanks @DeeJayElly 🎉 Looks good now!

Only this issue remains: https://github.com/input-output-hk/daedalus/pull/1871#issuecomment-596502232

**Comment by DeeJayElly** [comment]
> @DeeJayElly the state dir line breaks in an awkward way in Japanese:
> Screenshot 2020-03-09 at 13 41 36

Done

**Comment by nikolaglumac** [comment]
Great @DeeJayElly 🎉 

@a-rukin can you please check the spacing between the label and the "COPY" button? I believe it should be bigger than what it currently is... Please check and let @DeeJayElly know 🙏 

spacing

**Comment by DmitriiGaico** [comment]
@DeeJayElly Hey! As discussed please double check Linux fonts.
image

File: source/renderer/app/components/status/DaedalusDiagnostics.scss [hunk 1]
```diff
@@ -230,7 +224,11 @@
 =            SPECIFIC STYLES            =
 =======================================*/
 
-td.platformVersion {
+.layoutHeader.stateDirectoryPath {
+  width: 244px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/status/DaedalusDiagnostics.scss]: @DeeJayElly can we avoid hardcoding width here?

File: source/renderer/app/components/status/DaedalusDiagnostics.scss [hunk 2]
```diff
@@ -230,7 +224,11 @@
...
+}
+
+.layoutData.platformVersion {
   max-width: 435px;
```
Comment (nikolaglumac) [inline source/renderer/app/components/status/DaedalusDiagnostics.scss]: @DeeJayElly do we still need this max-width?

File: source/renderer/app/components/status/DaedalusDiagnostics.scss [hunk 3]
```diff
@@ -331,9 +300,17 @@ td.stateDirectoryPath {
     transition: 0.15s opacity;
     width: 11px;
   }
+
+  @media (min-width: 1001px) and (max-width: 1560px) {
```
Comment (nikolaglumac) [inline source/renderer/app/components/status/DaedalusDiagnostics.scss]: @DeeJayElly can we get rid of this media query?

**Comment by nikolaglumac** [comment]
> @DeeJayElly Hey! As discussed please double check Linux fonts.
> image

@DmitriiGaico I don't think this font issue is related to this PR. Please test again - could be some glitch on your end. Compare v2-integration build too 🙏 

cc @DeeJayElly

**Comment by a-rukin** [comment]
> Great @DeeJayElly 🎉
> 
> @a-rukin can you please check the spacing between the label and the "COPY" button? I believe it should be bigger than what it currently is... Please check and let @DeeJayElly know 🙏
> 
> spacing

@DeeJayElly can we make left padding to be at least 24px? so the button would be connected to the right part on any width.

**Comment by DmitriiGaico** [comment]
@DeeJayElly Hello! looks like it was my local glitch. The fonts in the latest build looks same as in V2. Thank you!
cc @nikolaglumac

File: source/renderer/app/components/status/DaedalusDiagnostics.scss (hunk 1 above)
Comment (DeeJayElly) [reply source/renderer/app/components/status/DaedalusDiagnostics.scss]: Well the solution with flexbox works but only if we have a width set on the left element inside the left/right columns on the Diagnostic screen and we need to have it here.

File: source/renderer/app/components/status/DaedalusDiagnostics.scss (hunk 2 above)
Comment (DeeJayElly) [reply source/renderer/app/components/status/DaedalusDiagnostics.scss]: Yes we need it in deed, parent element needs to have some kind of a limitation in order to know how to flex the child items.

File: source/renderer/app/components/status/DaedalusDiagnostics.scss (hunk 3 above)
Comment (DeeJayElly) [reply source/renderer/app/components/status/DaedalusDiagnostics.scss]: We need this max-width calculation only in the case where the text is breaking. I can remove it but then i am trying to force some width calculations on the resolutions where i don't need calculations at all.

**Comment by DeeJayElly** [comment]
> > Great @DeeJayElly 🎉
> > @a-rukin can you please check the spacing between the label and the "COPY" button? I believe it should be bigger than what it currently is... Please check and let @DeeJayElly know 🙏
> > spacing
> 
> @DeeJayElly can we make left padding to be at least 24px? so the button would be connected to the right part on any width.

Done

**Pull request merged**